Sources/SemanticIndex/UpdateIndexStoreTaskDescription.swift

```diff
@@ -320,6 +320,25 @@ package struct UpdateIndexStoreTaskDescription: IndexTaskDescription {
         // Fake an output path so that we get a different unit file for every Swift file we background index
         "-index-unit-output-path", uri.pseudoPath + ".o",
       ]
+    guard let buildSystemIndexStorePath = await self.buildSystemManager.initializationData?.indexStorePath else {
+      struct NoIndexStorePathError: Error {}
+      throw NoIndexStorePathError()
+    }
+    if let indexStorePath = args.firstIndex(of: "-index-store-path"), indexStorePath + 1 < args.count {
```

Comment:
`lastIndex` in case there's multiple? `lastIndexStorePathArgument` could also use `lastIndex` rather than the reverse loop it has now.

Reply:
Good idea.